openreview/venue_request/process/revisionProcess.py

```diff
@@ -591,6 +591,9 @@ def process(client, note, invitation):
                         )
                         client.post_note(comment_note)
 
+            if forum_note.content.get('accepted_submissions_only', '') == 'Enable revision for accepted submissions only':
+                conference.create_submission_revision_stage()
```

Comment:
shouldn't be this inside of the post_decision_stage implementation?

I think we should call to `create_submission_stage` to any existing submission revision invitation that is active, no matter is only for accepted papers, what do you think?

Reply:
I think I am worried that if only author names of accepted submissions are released, then re-running a submission revision stage that is enabled for all papers would remove readers from all revision invitations, even rejected papers. 

Also, once the post submission stage is run and accepted paper authors are revealed, we only allow the revision stage to be run for accepted papers: #2039